c/src/ml-api-inference-internal.c

```diff
@@ -297,6 +297,44 @@ __ml_validate_model_file (const char *const *model,
   return ML_ERROR_NONE;
 }
 
+/**
+ * @brief Internal helper function to read the string from specific ini file.
+ */
+static gchar *
+__ml_get_string_from_conf (const gchar * ini_path,
+    const gchar * group, const gchar * key)
+{
+  g_autofree gchar *value = NULL;
+  g_autoptr (GKeyFile) key_file = NULL;
+
+  if (!ini_path || !g_file_test (ini_path, G_FILE_TEST_EXISTS)) {
+    _ml_loge
+        ("The parameter, ini_path, is NULL. It should be a valid path of configuration file.");
```

Comment:
If it is not null and g_file_test fails, you need to show the given file path and say the file does not exist.